docs-2.0/20.appendix/0.FAQ.md

```diff
@@ -408,6 +408,14 @@ telnet: connect to address 192.168.1.10: Connection refused
 
 No, the licenses of Dashboard, Explorer, and Nebula Graph Enterprise Editions are independent of each other and cannot be used interchangeably.
 
+### During the validity period of the Nebula Graph Enterprise Edition license, after replacing the enterprise edition Meta with the community edition Meta, can the community edition Meta be used with the enterprise edition Graph and Storage?
```

Comment:
validity -> availible?

Reply:
> validity -> availible?

We use "validity period" to indicate the state of being legally acceptable and use "availability period" to indicate the state of being available. In this case, I think we can use both of them (availability period or validity period).